infra/base-images/base-builder/compile

```diff
@@ -42,19 +42,7 @@ then
   fi
 fi
 
-# Coverage flag overrides.
-COVERAGE_FLAGS_VAR="COVERAGE_FLAGS_${SANITIZER}"
-if [[ -n ${!COVERAGE_FLAGS_VAR+x} ]]
-then
-  export COVERAGE_FLAGS="${!COVERAGE_FLAGS_VAR}"
-fi
-
-# Don't need coverage instrumentation for engine-less builds.
-if [[ $FUZZING_ENGINE = "none" ]]; then
-  export COVERAGE_FLAGS=
-fi
-
-export CFLAGS="$CFLAGS $SANITIZER_FLAGS $COVERAGE_FLAGS"
```

Comment:
actually SANITIZER_FLAGS is no longer included after this deletion too.

Reply:
Good catch re `COVERAGE_FLAGS`! Those are still defined by base-builder/Dockerfile and might be overwritten by fuzzing engines `compile_*` scripts.

`SANITIZER_FLAGS` should still be there (weird GitHub UI moved them up to line 45).

Either way, I'm bringing this coverage flags override code back, as otherwise it looks confusing to have empty `COVERAGE_FLAGS` for a coverage build :D